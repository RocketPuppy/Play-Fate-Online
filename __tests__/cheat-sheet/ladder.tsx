import * as React from "react";
import * as renderer from "react-test-renderer";
import Ladder from "../../src/cheat-sheet/ladder";

test("renders all the pieces", () => {
  const render = renderer.create(<Ladder />);

  expect(render.toJSON()).toMatchSnapshot();
});
