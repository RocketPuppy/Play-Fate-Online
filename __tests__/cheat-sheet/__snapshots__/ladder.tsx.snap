// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    The Ladder
  </h2>
  <dl>
    <dt>
      +8
    </dt>
    <dd>
      Legendary
    </dd>
    <dt>
      +7
    </dt>
    <dd>
      Epic
    </dd>
    <dt>
      +6
    </dt>
    <dd>
      Fantastic
    </dd>
    <dt>
      +5
    </dt>
    <dd>
      Superb
    </dd>
    <dt>
      +4
    </dt>
    <dd>
      Great
    </dd>
    <dt>
      +3
    </dt>
    <dd>
      Good
    </dd>
    <dt>
      +2
    </dt>
    <dd>
      Fair
    </dd>
    <dt>
      +1
    </dt>
    <dd>
      Average
    </dd>
    <dt>
      +0
    </dt>
    <dd>
      Mediocre
    </dd>
    <dt>
      -1
    </dt>
    <dd>
      Poor
    </dd>
    <dt>
      -2
    </dt>
    <dd>
      Terrible
    </dd>
  </dl>
</div>
`;
